src/backend/base/langflow/components/vectorstores/astradb.py

```diff
@@ -607,6 +616,18 @@ def _add_documents_to_vector_store(self, vector_store) -> None:
                 msg = "Vector Store Inputs must be Data objects."
                 raise TypeError(msg)
 
+        if documents and self.deletion_field:
+            self.log(f"Deleting documents where {self.deletion_field}")
```

Comment:
nit: should we remove this log line?

Reply:
@msmygit I can see the argument to delete it for sure - i'd say we can keep it for now... I don't think it exposes any unnecessary information, and might help debug some issues.